### DISCUSSION TITLE
K3s Klipper LB with RKE2

### DISCUSSION THREAD

**Discussion by apcheamitru** [rancher/rke2#5787] [Q&A] [answered]
Is it possible to run the [Klipper Service Load Balancer](https://github.com/k3s-io/klipper-lb) with RKE2?  I'm running on a single node and am looking for a lightweight LoadBalancer solution that doesn't involve configuring a separate VIP.

**Comment by manuelbuil** [comment, marked as answer]
Yes, it is possible. The option is `enable-servicelb`

**Comment by hadisaboorimanesh** [comment]
how to  install Klipper on RKE2 ?

**Reply by brandond** [reply]
Read the comment directly above yours? It is also covered in the documentation.
